source/mentoringprogramme-icfoss-pilot.md

```diff
@@ -6,9 +6,9 @@ title: Pilot Mentoring Programme with India ICFOSS
 > happened back in 2013.
 
 The ASF has been participating in many mentoring initiatives, and it's now
-partening in a pilot project with India ICFOSS to provide mentoring for 
-undergraduated and graduated students that have interest in learning how
-to participate in open source communitites at ASF.
+partnering in a pilot project with India ICFOSS to provide mentoring for
+undergraduate and graduated students that have interest in learning how
```

Comment:
probably should also s/graduated/graduate/

Reply:
indeed. Missed that one.